scripts/python_scripts/generate_network_status.py

```diff
@@ -35,65 +36,55 @@
 
 
 def generate_networks_list():
+    sub_query = SubQueryDeploymentAPI(auth_token=token, org=organisation)
+    sub_query.collect_all_data()
+
     writer = MarkdownTableWriter(
         headers=["--", "Network", "Features", "Stage status",
                  "Prod status", "Stage commit", "Prod commit"],
-        value_matrix=generate_value_matrix(),
+        value_matrix=generate_value_matrix(sub_query),
         margin=1
     )
     writer.write_table()
+    asyncio.run(send_telegram_message(telegram_message))
```

Comment:
Didn't get why we are sending error message to telegram while generating network list.